app/ts/background/background.ts

```diff
@@ -158,7 +158,7 @@ export async function refreshConfirmTransactionSimulation(
 }
 
 // returns true if simulation state was changed
-export async function getPrependTrasactions(ethereumClientService: EthereumClientService, settings: Settings, richMode: boolean) {
+export async function getPrependTrasactions(ethereumClientService: EthereumClientService, settings: Settings, richMode: boolean): Promise<WebsiteCreatedEthereumUnsignedTransaction[]> {
```

Comment:
oh, I have never heard of `satisfies` keyword. I think I could find that useful in some places 🤔 

I often wants to specify the return type of function, as sometimes I am crafting the complex type and then its nice to see that I have constructed the type correctly as I get error otherwise